After flatpak update GPU inaccessible in Flatpak.

Everything was working flawlessly up until today when i decided to update flatpaks...  Launcher/ runelite works as expected but when i try to launch rs3 client i get an error message... 

![Screenshot at 2023-12-05 18-09-29](https://github.com/USA-RedDragon/jagex-launcher-flatpak/assets/139579998/a76b7815-1c19-45cb-a5b2-e0221ce09ab3)


[System]
OS:              Artix Linux -openrc
Arch:            x86_64
Kernel:          6.6.3-artix1-1
Desktop:         MATE
Display Server:  x11

[CPU]
Vendor:          AuthenticAMD
Model:           AMD Ryzen 7 5700G with Radeon Graphics
Physical cores:  8
Logical cores:   16

[Memory]
RAM:             30.6 GB
Swap:            8.0 GB

[Graphics]
Vendor:          AMD
OpenGL Renderer: AMD Radeon Graphics (renoir, LLVM 16.0.6, DRM 3.54, 6.6.3-artix1-1)
OpenGL Version:  4.6 (Compatibility Profile) Mesa 23.2.1-arch1.2
OpenGL Core:     4.6 (Core Profile) Mesa 23.2.1-arch1.2
OpenGL ES:       OpenGL ES 3.2 Mesa 23.2.1-arch1.2
Vulkan Version:  1.3.269
Vulkan Drivers:  AMD Radeon Graphics (RADV RENOIR) (1.3.255)

I can reproduce this issue with the RS3 client on a Steam Deck. But, I'm also getting a similar issue when trying to start HDOS:

```
java.lang.RuntimeException: Exception while creating the OpenGL context
	at org.lwjgl.opengl.awt.AWTGLCanvas.beforeRender(AWTGLCanvas.java:81)
	at hdos.th.T(th.java:14)
	at hdos.th.P(th.java:2)
	at hdos.Main.main(Main.java:132)
Caused by: java.awt.AWTException: No supported framebuffer configurations found
	at org.lwjgl.opengl.awt.PlatformLinuxGLCanvas.create(PlatformLinuxGLCanvas.java:68)
	at org.lwjgl.opengl.awt.PlatformLinuxGLCanvas.create(PlatformLinuxGLCanvas.java:121)
	at org.lwjgl.opengl.awt.AWTGLCanvas.beforeRender(AWTGLCanvas.java:79)
	... 3 more
```

@MushyMarks, can you confirm if HDOS has that same error when launching?

RuneLite works fine, but I think that uses software rendering.

--- a/com.jagex.Launcher.yaml
+++ b/com.jagex.Launcher.yaml
@@ -23,8 +23,6 @@ finish-args:
   - --env=WINEARCH=win64
   - --env=WINEDLLOVERRIDES=dxgi=b # Fix launch on Steam Deck by using builtin dxgi
   - --env=GST_PLUGIN_SYSTEM_PATH=/app/lib32/gstreamer-1.0:/app/lib/gstreamer-1.0:/usr/lib/i386-linux-gnu/gstreamer-1.0:/usr/lib/x86_64-linux-gnu/gstreamer-1.0
-  - --env=mesa_glthread=true # https://gitlab.freedesktop.org/mesa/mesa/-/blob/94d827332f39f537debde50125e3f4f9f5fd34c4/src/gallium/frontends/dri/dri_context.c#L221
-  - --env=MESA_LOADER_DRIVER_OVERRIDE=zink # https://docs.mesa3d.org/envvars.html#envvar-MESA_LOADER_DRIVER_OVERRIDE
 sdk-extensions:
   - org.freedesktop.Sdk.Compat.i386
   - org.freedesktop.Sdk.Extension.openjdk11

